Mention tsh proxy db --tunnel in per-session MFA docs (#65816)
gravitational/teleport: docs/pages/includes/database-access/proxy-db-tunnel.mdx

{{ db="aurora" dbArgs="" }}

<Admonition type="tip">

> To connect to the database using a tool of your choice, like a graphical
> database client, you can create a local tunnel and point it to your software:

Reviewer: ```suggestion
To connect to the database using a tool of your choice (for example, a graphical database client), create a local tunnel and configure your client to use it.
```

> {{ db="aurora" dbArgs="" }}

Reviewer 2: nit: do we want to give a quick mention to Teleport Connect? UX is better than `tsh proxy db` imo

Author: Can we address it separately? I'm not familiar with that UX, and mentioning the proxy flow was a secondary task to my original goal anyway, which was clarifying how the proxy works with per-session MFA.

<Admonition type="tip">
To connect to the database using a tool of your choice (for example, a
graphical database client), create a local tunnel and configure your client to
use it.

```code
$ tsh proxy db {{ db }} --tunnel {{ dbArgs }}
```

> See [Database GUI Clients](../../connect-your-client/third-party/gui-clients.mdx#sql-server-with-azure-data-studio)

Reviewer 2: ```suggestion
See [Database GUI Clients](../../connect-your-client/third-party/gui-clients.mdx)
```
